app/src/main/java/com/example/news/data/local/Converter.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.news.data.local
+
+import com.example.news.model.Article
+import com.example.news.model.CachedArticle
+import com.example.news.model.CachedData
+import com.example.news.model.NewsApi
+
+//class Converter {
```

Comment:
FYI you still need the converter if you are going to insert a list into room

Reply:
done